.github/workflows/server-test-template.yml

```diff
@@ -44,9 +44,7 @@ jobs:
         env:
           BUILD_IMAGE: mattermostdevelopment/mattermost-build-server:${{ steps.go.outputs.GO_VERSION }}
         run: |
-          if [[ ${{ github.ref_name }} == 'master' ]]; then
-            export RACE_MODE="-race"
-          fi
+          export RACE_MODE="-race"
```

Comment:
I guess you're going to revert this after the review?

Reply:
Yep